### PR TITLE
using setmetatable for locales

### PR DESCRIPTION
**Pull request by Cyanokaze**
so you can using L["Key"] form for any string in anywhere and don't need define it first.
Also, add Traditional Chinese part.

### CODE REVIEW

**Comment by Cyanokaze** [comment]
Addon Locale's general practice

File: localization.lua [hunk 1]
```diff
@@ -11,102 +11,14 @@ local L = setmetatable({}, {
...
-	L["Multi-Shot"]="Multi-Shot"
-	L["Aimed Shot"]="Aimed Shot"
-	L["Shoot"]="Shoot"
-	L["Quick Shots"]="Quick Shots"
-	L["Rapid Shot"]="Rapid Shot"
-	L["Berserking"]="Berserking"
-	L["Kiss of the Spider"]="Kiss of the Spider"
-	L["Curse of Tongues"]="Curse of Tongues"
-]]
+	L["Thank you for installing WeaponSwingTimer Version"] = true
```
Comment (watchyoursixx) [inline localization.lua]: Not sure what this is.

**Review by watchyoursixx — COMMENTED** [review]
What does the L["Thank you for installing WeaponSwingTimer Version"] = true do? Does it ever get set to false with different locales?

**Comment by Cyanokaze** [comment]
> What does the L["Thank you for installing WeaponSwingTimer Version"] = true do? Does it ever get set to false with different locales?

true means nothing. if L[key] is not exists, will return key as value. And if value is true, also return key as value.

**Pull request merged**